packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -4815,6 +4921,7 @@ class Row extends Flex {
   /// must not be null.
   const Row({
     super.key,
+    super.selectionSeparator,
```

Comment:
I don't think we need to expose this for now. I can't think of a good use case where they would want to change this. We can open this up if we have real use case

Reply:
Removed